### DISCUSSION TITLE
Equations: Multiple Alignments per Line (&& in LaTeX)

### DISCUSSION THREAD

**Discussion by nu444** [typst/typst#2646] [Q&A] [answered]
HI!
I often need to annotate individual lines of multiline equations. In LaTeX, the align* environment uses & as the main alignment indicator, and && as a seperate alignemt for annotations, which are also spaced out a bit from the equation. Is there a way to do this in Typst? Just using & doesn't seem to work.
Thanks!

An example of what it looks like in LaTeX:
![Bildschirmfoto 2023-11-10 um 23 34 50](https://github.com/typst/typst/assets/118182268/3df1a9ed-b1a1-4a0d-8dcf-03caec3fd8d8)

**Comment by Enivex** [comment, marked as answer]
It works exactly like alignat in latex. That is, exactly like align, except there's no extra space between pairs of columns. You have to add that manually for the time being.

**Reply by recmo** [reply]
How do I add extra space?
